Assets/MixedRealityToolkit-Examples/UX/Scripts/Controls/SliderGestureControl.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
-using UnityEngine.Events;
+using HoloToolkit.Unity;
```

Comment:
Please check namespaces, as these were all updated to MixedRealityToolkit

Reply:
This refers to the namespace where the **UnityEventFloat** class is located (added as part of this PR).  Should this class use a different namespace (it matches those present alongside it)?

Reply:
I did not notice that the namespace had changed in dev.  Should I change the namespace? Or should a minor change like this target the stabilisation branch? (This is my first PR).  Advice welcome.

Reply:
Best bet is to retarget the PR for the stabilization branch if you want to see these fixes in the next release

Reply:
OK, thanks.